{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "@allure.story(\"中断功能\")\n    @allure.severity(allure.severity_level.BLOCKER)\n    def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\ntest_allure.py:27: Exception"}, "start": 1659526070722, "stop": 1659526070722, "uuid": "61e5515f-875d-41df-8573-65d7aa076a2d", "historyId": "985c3d972d231f87d0914b89f78754fd", "testCaseId": "cb41bd139e929d9eef92825c451cc27a", "fullName": "test_allure#test_broken", "labels": [{"name": "severity", "value": "blocker"}, {"name": "story", "value": "中断功能"}, {"name": "suite", "value": "test_allure"}, {"name": "host", "value": "yin"}, {"name": "thread", "value": "18128-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_allure"}]}